query_sheet.rb

```diff
@@ -389,17 +390,17 @@ def candidates_to_move
     candidates_with_invalid_ads = []
     candidates = existing_candidates.reduce({}) do |candidates, (id, candidate)|
       current_ad, current_ed = candidate.values_at('AD', 'ED')
-      unless candidate['Address'].match(
-        /\d+ \w+.*,? brooklyn,? ny,? 112\d{2}([-—]\d{4})?$/i
-      )
-        candidates_with_invalid_addresses << values_and_type(
-          INVALID_ADDRESSES_SPREADSHEET_ID,
-          'existing candidates',
-          candidate.values,
-          type
-        )
-        next candidates
-      end
+      # unless candidate['Address'].match(
+      #   /\d+ \w+.*,? brooklyn,? ny,? 112\d{2}([-—]\d{4})?$/i
+      # )
+      #   candidates_with_invalid_addresses << values_and_type(
+      #     INVALID_ADDRESSES_SPREADSHEET_ID,
+      #     'existing candidates',
+      #     candidate.values,
+      #     type
+      #   )
+      #   next candidates
+      # end
```

Comment:
this was commented out on my machine, i believe from our (home-projected) group code review 9/22/21, so i kept it commented out—iirc, we didn't want to filter out addresses that weren't explicitly brooklyn addresses, as they oftentimes actually _were_  brooklyn addresses, albeit poorly-formatted, after all?